README.md

```diff
@@ -142,26 +142,23 @@ BOLOS_SDK=$NANOS_SDK make DEBUG=1
 You can replace `NANOS` with `NANOSP`, `NANOX`, `STAX` for the other devices in BOLOS_SDK environmental variable.
 
 ### Testing
-To test the application you need to enable python virtual environment and some dependencies. On any operating system, create a python virtual environment and activate it.
+The application tests are run using same docker container used for building. Inside the docker container run following script,
 ```
-$ sudo apt-get update && sudo apt-get install -y qemu-user-static tesseract-ocr libtesseract-dev python3-pip libgmp-dev libsodium-dev git
-$ python3 -m venv env
-$ source env/bin/activate
-```
-Inside the virtualenv, load the requirements.txt file.
-```
-(env)$ pip install -r test/requirements.txt
+$ apk add gmp-dev curl jq libsodium-dev git xxd procps;
```

Comment:
`curl`, `jq`, `git`, `xxd` and `procps` and not required

Reply:
Removed.